common/persistence/cassandra/cassandraClusterMetadata.go

```diff
@@ -148,3 +163,49 @@ func (m *cassandraClusterMetadata) GetImmutableClusterMetadata() (*p.InternalGet
 		ImmutableClusterMetadata: p.NewDataBlob(immutableMetadata, common.EncodingType(encoding)),
 	}, nil
 }
+
+func (m *cassandraClusterMetadata) GetActiveClusterMembers(request *p.GetActiveClusterMembersRequest) (*p.GetActiveClusterMembersResponse, error) {
+	query := m.session.Query(templateGetActiveClusterMembership, time.Now().UTC().Add(-request.LastHeartbeatWithin))
+
+	// No paging enabled due to TTL on table
```

Comment:
I'm confused about this comment here.  How is TTL related to pagination?  Actually it would be good if we support pagination for the very beginning.